e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -254,3 +293,13 @@ function publishChanges(callback) {
     callback && callback(targetXhr);
   });
 }
+
+function setParameter(name, filter) {
+  cy.findByText("Which parameters can users of this embed use?")
+    .parent()
+    .within(() => {
+      cy.findByText(name).siblings("a").click();
+    });
```

Comment:
I presume this should also work

```suggestion
  cy.findByText("Which parameters can users of this embed use?")
    .parent()
    .findByText(name).siblings("a").click();
```